CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # @digitalbazaar/http-client ChangeLog
 
+## 2.0.0 -
+
+### Changed
+- **BREAKING**: No longer includes full URL in `Possible CORS error.` type errors.
+  Instead, only includes the request's `host`.
```

Comment:
If the error does not include the path, how does one go about troubleshooting this sort of error?  Many errors here will be more difficult to troubleshoot without knowing the endpoint that is being contacted.

I think I would be in favor of keeping the `path` component of these errors, but redacting `query` parameters if they exist.  I'm interested in getting other opinions before you roll back any changes here.

Reply:
@mattcollier - yeah, I was pondering about that too.
I can see it either way, so we should just decide our policy.

Reply:
Of course, if the URL is a "capability URL" (it gives you some kind of privilege just by knowing it) it's not going to be safe to log the path either. Maybe we include the path based on environment variables? In the browser, we can include the full URL, in node if the environment is production we only include the host? Thoughts?

Reply:
That seems like a decent compromise. (Although I'm not sure if we have precedent/experience with using ENV vars for isomorphic libs)